File: src/arte/programar/ejercicio02/java/Main.java
package arte.programar.ejercicio02.java;

import java.util.Scanner;

public class Main {
    /**
     * Pedir dos números por consola e imprimir si son iguales o no.
     * @param args
     */
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);

        try {
            System.out.print("Ingrese Numero 1: ");
            int n1 = console.nextInt();

            System.out.print("Ingrese Numero 2: ");
            int n2 = console.nextInt();

            if (n1 == n2) {
                System.out.println("Los numeros son iguales.");
            } else {
                System.out.println("Los numeros no son iguales");
            }
        } catch (Exception ex){
            System.out.println("Error : " + ex.getMessage());
        }
    }
}

File: src/arte/programar/ejercicio02/kotlin/Main.kt
package arte.programar.ejercicio02.kotlin

import java.util.*

object Main {
    /**
     * Pedir dos números por consola e imprimir si son iguales o no.
     * @param args
     */
    @JvmStatic
    fun main(args: Array<String>) {
        val console = Scanner(System.`in`)

        try {
            print("Ingrese Numero 1: ")
            val n1 = console.nextInt()

            print("Ingrese Numero 2: ")
            val n2 = console.nextInt()

            if (n1 == n2) {
                println("Los numeros son iguales.")
            } else {
                println("Los numeros no son iguales")
            }
        } catch (ex: Exception) {
            println("Error : " + ex.message)
        }
    }
}